xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -50,6 +50,8 @@ class CRenderManager;
 class IRenderMsg
 {
   friend CRenderManager;
+public:
+  virtual ~IRenderMsg() = default;
```

Comment:
I guess you are right for interfaces (== classes with only pure virtual member functions). Will remove the respective changes from this PR.